R/meta_regression/cov_run_model.R

```diff
@@ -6,15 +6,8 @@
 covariate_run_model_ui <- function(id) {
   ns <- NS(id)
   div(
-    helpText(
-      "Baysesian covariate meta-regression using the gemtc package.",
-      br(),
-      "Heterogeneity prior: standard deviation ~ U(0,X), where X represents a ",
-      tags$i("very large"),
-      "difference in the analysis' outcome scale and is determined from the data.",
-      br(),
-      tags$strong("Please note each simulation may take 20 seconds.", style = "color:#FF0000")
-    ),
+    uiOutput(ns("model_text")),
```

Comment:
How would you feel about changing this to use `textOutput` instead?

I'm thinking that the name of the package could be passed into the UI function, the long runtime warning could be standardised to 60s, and the prior could become a textOutput.

I dislike `uiOutput` in general because it makes the code harder to follow, so using a more specific output is cleaner.

Reply:
I suspected this comment was coming, because you said the same thing in your review of Naomi's graph.
It's done, but we've lost the italics on "very large". I don't think textOutput can do italics from what I've read.